conf/defaults.ini

```diff
@@ -230,6 +230,9 @@ snapshot_remove_expired = true
 # Number dashboard versions to keep (per dashboard). Default: 20, Minimum: 1
 versions_to_keep = 20
 
+# Minimum dashboard refresh interval
+min_refresh_interval = 5m
```

Comment:
This is a breaking change, is that by accident since I see in settings.go the default is empty string?

Reply:
Thanks for spoting this ! I removed it now.